api/core/tools/provider/builtin/websearch/tools/news_search.yaml

```diff
@@ -98,11 +98,11 @@ parameters:
     type: select
     label:
       en_US: Country
-      zh_Hans: 国家
+      zh_Hans: 国家/地区
     required: false
     human_description:
       en_US: Defines the country of the search. Default is "US".
-      zh_Hans: 定义搜索的国家/地区。默认为“美国”。
+      zh_Hans: 定义搜索的国家/地区/地区。默认为“美国”。
```

Comment:
This line shouldn't be changed; the previous version was correct.

Reply:
updated